docs/source/release/v6.0.0/indirect_geometry.rst

```diff
@@ -5,33 +5,35 @@ Indirect Geometry Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
 New Features
 ############
-- Added the ability to save a detector grouping to a XML file from a Custom Grouping String on Indirect Data Reduction. 
+- A Custom Grouping String in Indirect Data Reduction can now be saved to an XML file.
 
 Improvements
 ############
-- Added an option to load diffraction data on IN16B.
+- Diffraction data can now be loaded on IN16B.
 - Indirect Data Analysis multiple input tabs can now accept different fit ranges for each spectra.
-- :ref:`QENSFitSequential <algm-QENSFitSequential>` now accepts multiple inputs for `StartX` and `EndX` for each spectra.
-- Plotted fits in indirect data analysis are cleared when the fit parameters are changed.
-- The context menu for plots in the  indirect data analysis GUI is now enabled.
-- The function browser in the ConvFit tab should now correctly show Q values for Q dependent functions.
-- The ConvFit tab in the Indirect Data Analysis GUI will now set default parameters for the FWHM when a resolution file is loaded.
-- QENSFitSequential and PlotPeakByLogValue can accept a different mask range for each spectra via the `ExcludeMultiple` Parameter.
-- Tabs in the Indirect Corrections interface not has a scroll bar to prevent being squashed on small screens.
+- :ref:`QENSFitSequential <algm-QENSFitSequential>` can now accept different inputs for `StartX` and `EndX` for each
+  spectra.
```

Comment:
Four versions of each spectra on this page, which I didn't notice but MS Word did!
```suggestion
  spectrum.
```